Makefile

```diff
@@ -1,14 +1,14 @@
-NAME            := mpifileutils
-SRC_EXT         := gz
-TEST_PACKAGES   := $(NAME)-mpich-devel $(NAME)-openmpi3-devel
-REPO_NAME       := mpifileutils-pkg
-SOURCE_COMMIT   := 7c32b9cf6143776d798ab8134696b5f4e38916e9
-SOURCE          = $(SOURCE_COMMIT).tar.$(SRC_EXT)
-GIT_SHORT       := $(shell git rev-parse --short $(SOURCE_COMMIT))
-BUILD_DEFINES   := --define "%relval .g$(GIT_SHORT)" --define "%source_commit $(SOURCE_COMMIT)"
+NAME       := mpifileutils
+SRC_EXT    := gz
+REPO_NAME  := mpifileutils-pkg
+
+GIT_SHORT       := $(shell git rev-parse --short=8 HEAD)
+GIT_NUM_COMMITS := $(shell git rev-list HEAD --count)
+
+BUILD_DEFINES   := --define "%relval .$(GIT_NUM_COMMITS).g$(GIT_SHORT)"
```

Comment:
```suggestion
BUILD_DEFINES   := --define "relval .$(GIT_NUM_COMMITS).g$(GIT_SHORT)"
```

Reply:
I can see why this fails, but I don't understand how the same things works here:
https://github.com/daos-stack/daos/blob/8c754728adf7c05a3e59b911c454e668026cd20f/utils/rpms/Makefile#L16